cristin-project/src/test/java/no/unit/nva/cristin/projects/query/projectQuery.feature

```diff
@@ -100,7 +100,7 @@ Feature: API tests for Cristin projects query
     And match contentType == PROBLEM_JSON_MEDIA_TYPE
     And match response.title == 'Bad Request'
     And match response.status == 400
-    And match response.detail == "Invalid query parameter supplied. Valid parameters: ['language', 'organization', 'page', 'query', 'results', 'status']"
```

Comment:
I did not see this until now, but maybe we can consider using " as it was instead of ' to avoid escaping ' in the string itself. But its not a must and we can have it as is